genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/ClusterTask.java

```diff
@@ -95,6 +98,13 @@ public void executeTask(@NotNull final Map<String, Object> context) throws Genie
                 AdminResources.CLUSTER
             );
 
+            // Create the dependencies directory for this id
+            createEntityInstanceDependenciesDirectory(
```

Comment:
Not sure what you are asking to do differently.
I am calling an existing method in BaseTask that has (admittedly very simple) logic to create dependencies folders.